triggermail_templates.py

```diff
@@ -128,6 +134,53 @@ def dissect_filename(self, template_filename):
         self.partner = self.settings.get("partner", self.partner) or self.partner
         self.partner = self.partner.replace("_templates", "")
 
+    def generate_pertinent_file_names_and_map(self, template_filename):
+        template_filename = template_filename.replace(self.path + '/', '')
+        image_regex = "/img/" + self.partner + "/" + "(.*?)['\"]"
+        file_map = dict()
+        all_file_names_required = [template_filename]
+        all_images_required = []
+        # go through all files this template requires
+        # we get all the filenames, their contents, and images
+        for file_name in all_file_names_required:
```

Comment:
Can you get rid of this loop? It only ever contains `template_filename`? OH; you modify this list below. I see. That works? okay then never mind